test/source-sink-errors/mzcompose.py

```diff
@@ -35,12 +41,90 @@ def run_test(self, c: Composition) -> None:
         ...
 
 
+@dataclass
+class BadKafkaDisruption:
+    name: str
+
+    def run_test(self, c: Composition) -> None:
```

Comment:
```suggestion
    // override the `run_test`, as we need `Kafka` (not `Redpanda`), and need to change some other things
    def run_test(self, c: Composition) -> None:
```

Reply:
Note: it's overriding `run_test` of `Disruption` not `KafkaDisruption`